src/new-components/buttons/button.tsx

```diff
@@ -25,29 +27,32 @@ const StyledButton = styled.button`
 `;
 
 const StyledIcon = styled(Icon)`
-  padding-right: 10px;
-  // pull the fixed size icon closer to the text
-  margin-left: -4px;
-  /**
-    This is a magic number to match the vertical middle of the button
-    with the icon which has a fixed bounding box which won't allow us to align
-    it based on the actual shape of the icon.
-   */
-  vertical-align: -18%;
+  margin-right: 10px;
 `;
 
 const ButtonText = styled.span`
+  ${TextStyles.toplineR};
+  line-height: 24px;
   padding-left: 16px;
-  padding-right: 16px;
+  padding-right: 6px;
 `;
 
-type RegularButtonProps = React.ComponentPropsWithoutRef<'button'>;
-
-export const Button = (props: RegularButtonProps) => {
-  return (
-    <StyledButton {...props}>
-      <ButtonText>{props.children}</ButtonText>
-      <StyledIcon show={'chevron_right'} size={IconSize.NORMAL} />
-    </StyledButton>
-  );
+type RegularButtonProps = React.ComponentPropsWithoutRef<'button'> & {
+  to?: string;
 };
+
+export const Button = (props: RegularButtonProps) => (
+  <>
+    {props.to ? (
```

Comment:
I'm not reading anything about that behavior in the story. Could you explain the fact in a section "Button vs. Anchors`. In addition to a few words by you, you can also link to some article explaining the difference. I didn't know the following article before but it looks a great explanation of this topic https://bitsofco.de/anchors-vs-buttons/

Reply:
I usually remember "anchor for links, button to trigger interactions" and that button support enter key presses by default which is important for a11y but also UX.